Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs

```diff
@@ -511,6 +525,16 @@ public void ZitiResponse(object evt)
 		public void ZitiDumpToFile(string fileName) {
 			NAPI.z4d_ziti_dump_file(this.WrappedContext.nativeZitiContext, fileName);
 		}
+
+		private async Task<string> GetControllerVersionAsync(string ztAPI)
+		{
+			string versionUrl = "version";
+			string versionReponse = await OpenZiti.WebClient.HttpGet(ztAPI, versionUrl);
```

Comment:
this is already captured for us in the C SDK I thought? Why make a web request?

Reply:
`ziti_get_controller_version` that should return this information. Is it not working when you use it?

Reply:
I will test this function and let you know